### ISSUE TITLE
Operation after a variable assignement broke syntax coloration

### ISSUE THREAD

**Issue by Poil** [rodjek/vim-puppet#73]
```
    $toto = $fact / 1024
    $titi
```
$titi is wrong colored, I'm unable to find why for the moment

**Comment by Poil** [comment]
The error is here 

```
"syn region  puppetRegex            start="/" skip="\\/" end="/" contains=puppetRegexParen,puppetRegexBrace,puppetRegexOrpuppetRegexBrack,puppetRegexComment
```

**Comment by alexjfisher** [comment]
@Poil Did you ever find a fix?

**Comment by Poil** [comment]
No sorry

**Comment by mandos** [comment]
This issue is duplicates of #46, @GerbenWelter proposed some simple fix for it.

**Comment by GerbenWelter** [comment]
This should be fixed by f200678 now.
